### DISCUSSION TITLE
Add additional views within Rodauth

### DISCUSSION THREAD

**Discussion by atthemiko** [janko/rodauth-rails#164] [Q&A]
Hi Janko

Is it possible to add additional routes and views within Rodauth? 

somewhat like this perhaps

```
route do |r|
    r.rodauth # route rodauth requests
    r.get("sign-up-confirm"){  render path/to/views/rodauth/sign-up-confirm.html.erb }
 end
```

**Comment by janko** [comment]
Yes, but in this case you could just use classic Rails, for example:

```rb
# config/routes.rb
Rails.application.routes.draw do
  get "sign-up-confirm", to: "rodauth#sign_up_confirm"
end
```
```rb
# app/controllers/rodauth_controller.rb
class RodauthController < ApplicationController
  def sign_up_controller
    render "sign-up-confirm"
  end
end
```

Note that you don't need to use `RodauthController`, you can also put these routes in another controller.

**Reply by atthemiko** [reply]
Yup that's my current workaround. Was wondering if it's possible within Rodauth to keep it centralised and clean.

Thanks anyway!
